sphinx_gherkindoc/writer.py

```diff
@@ -33,7 +33,6 @@
 AVAILABLE_ROLES = [
     "gherkin-step-keyword",
     "gherkin-step-content",
-    "gherkin-step-text",
```

Comment:
NB: Man it'd be cool if this list could be used to generate the one in the README somehow.
I am not sure how we would do that (I can think of many ways, but none I want to suggest without playing first).
So this is really a kind of meta-NB I guess.